code/modules/mob/living/carbon/human/life.dm

```diff
@@ -64,6 +64,19 @@
 	if(has_trait(TRAIT_PACIFISM) && a_intent == INTENT_HARM)
 		to_chat(src, "<span class='notice'>You don't feel like harming anybody.</span>")
 		a_intent_change(INTENT_HELP)
+		
+	GET_COMPONENT_FROM(mood, /datum/component/mood, src)
+	if (getBrainLoss() >= 60 && stat == CONSCIOUS)
```

Comment:
why are you checking this every tick

Reply:
why the hell are you pinging me about this

you did this

Reply:
what no i didnt

Reply:
Did i?

Reply:
@ike709 i checked the code is bad so it must be yours

Reply:
fixed it a tiny bit but besides that it doesnt matter its checked every tick because it returns if the event already exists